### PR TITLE
#14 Fix placeholder

### LINKED ISSUE #14: Placeholder does not hold place
# Current behavior

i14

### PR DESCRIPTION
**Pull request by RyoNkmr**
this closes #14 

# Changes

## Before

i14

## After

i14fixed

### CODE REVIEW

File: src/sticky.js [hunk 1]
```diff
@@ -107,11 +110,19 @@ export default class Sticky {
...
 
-  static normalizeElement(value: string|HTMLElement, fallback: HTMLElement): HTMLElement {
+  static normalizeElement(value: SelectorOrElement, ...fallbacks: MaybeHTMLElement[]): HTMLElement {
     if (value instanceof HTMLElement) {
       return value;
     }
-    return document.querySelector(value) || fallback;
+
+    const element: ?HTMLElement = ([document.querySelector(value), ...fallbacks]
+      .find(item => !!item && item instanceof HTMLElement): any);
```
Comment (RyoNkmr) [inline src/sticky.js]: 👀 `Array.prototype.[filter|find]` type refinement problem workaround
https://github.com/facebook/flow/issues/1414#issuecomment-231680334

😭

Comment (resessh) [reply src/sticky.js]: get the power of Typescript 🙄

File: src/sticky.js [hunk 2]
```diff
@@ -1,11 +1,14 @@
 /* @flow */
 import Placeholder from './placeholder';
 
+type MaybeHTMLElement = HTMLElement|Element|null|void;
```
[the remark this answers is not in the document]
Comment (RyoNkmr) [reply src/sticky.js]:
```
Node.parentElement: ?Element;
```
https://github.com/facebook/flow/blob/v0.75.0/lib/dom.js#L439

parentElement is optional.

[the remark this answers is not in the document]
Comment (resessh) [reply src/sticky.js]: understood 👍

**Pull request merged**